lib/pbench/cli/agent/commands/cleanup.py

```diff
@@ -19,7 +18,7 @@ def execute(self):
             "%s deprecated, will be removed in future release in favor of pbench-clear-results",
             self.name,
         )
-        run_command("pbench-clear-results")
+        os.execlp("pbench-clear-results", "pbench-clear-results")
```

Comment:
OK, that's uglier than I hoped; but, yeah "path" and "arguments" and we need `argv[0]` to be set manually? Blech. 😬

Reply:
Still better than `run_command`!!!

Reply:
Yep, I had both of those thoughts, too.